serverless/functions/create-stream.js

```diff
@@ -293,44 +294,46 @@ module.exports.handler = async (context, event, callback) => {
     return callback(null, response);
   }
 
-  const conversationsClient = client.conversations.services(CONVERSATIONS_SERVICE_SID);
+  if (DISABLE_CHAT !== 'true') {
```

Comment:
Does this syntax work in JS?
```
if (!DISABLE_CHAT) {
```

Reply:
Or maybe it doesn't because this var is a string? But JS can automatically convert to bool right?

Reply:
We can retrieve environment variables in Node.js by accessing the `process.env` object. For some reason, the `process.env` object forces all of its properties to be of the type string. I am pretty sure Twilio Functions appends the `context` object to `process.env`, which is why when chat is disabled, `DISABLE_CHAT` is the string `'true'` instead of the boolean `true`.

I think if we wanted to convert the variable to a boolean I'd have to create a new variable and do something like:
```
let chat_enabled = true;
if (DISABLE_CHAT === 'true') {
     chat_enabled = false
}
```
But I could be wrong, @timmydoza what do you think?

Reply:
Oh interesting, yeah probably best the way it is.